design/noobaa/system-create.md

```diff
@@ -0,0 +1,20 @@
+♜ [Rook NooBaa Design](README.md) /
+# System Create
+
+The operator watches for NooBaa System CRD creations and in reaction deploys a new system with the following steps:
```

Comment:
Is it supported to create multiple instances of Noobaa? For example, could I create multiple noobaa operators that each manage independent systems? Or create multiple systems for the same operator to manage? Or is it expected to have a single system per K8s cluster? 

For example, most ceph clusters only have a single instance, but it is possible to create independent instances. The critical thing for ceph is that each cephcluster is created in an independent namespace to avoid the resources clashing with the same names and to keep things simpler.

Reply:
Since there doesn't seem to be any clashing in this case my plan is to support:
1. multiple systems per cluster
2. multiple systems per namespace
3. multiple systems per operator

If this will become a challenge I will only insist to support multiple systems per cluster, but I think it should be a similar effort to support all three.

Resolving.